api/src/main/java/org/openmrs/module/santedb/mpiclient/api/impl/FhirMpiClientServiceImpl.java

```diff
@@ -194,8 +192,45 @@ else if("basic".equals(this.m_configuration.getAuthenticationMode())) {
     @Override
     public List<MpiPatient> searchPatient(String familyName, String givenName, Date dateOfBirth, boolean fuzzyDate,
```

Comment:
Does this function have a unit test? If not, do you think we could quickly add one to test out the `otherDataPoints` parameter?